CMakeLists.txt

```diff
@@ -635,7 +635,7 @@ if (NOT Windows)
 	set(ATOMICS_TEST_SOURCE [=[
 		#include <atomic>
 		#include <cstdint>
-		std::atomic<int> x{0};
+		std::atomic<int> x(0);
```

Comment:
hi @FranciscoPombal, there is something slightly different with AppleClang 12.0.0.12000032, however, I can't find any reference to it.